packages/components/src/components/Map.tsx

```diff
@@ -144,10 +154,7 @@ export function Map({
         map.target.fitBounds(layerToZoomBounds);
       }}
     >
-      <TileLayer
-        attribution='&copy; <a href="https://www.openstreetmap.org/copyright">OpenStreetMap</a> contributors'
-        url="https://{s}.tile.openstreetmap.org/{z}/{x}/{y}.png"
-      />
+      <TileLayer url={tile.url} attribution={tile.attribution} {...tile.data} />
```

Comment:
Based on which provider was chosen, we can pass different props to this component.

I think you can do an implementation based on this code https://github.com/leaflet-extras/leaflet-providers/blob/master/leaflet-providers.js

Reply:
implemented this approach in latest commit
cc @demenech